nodes/scheduling/descheduler/nodes-descheduler-release-notes.adoc

```diff
@@ -30,6 +56,8 @@ The following advisory is available for the {descheduler-operator} 5.0.1:
 include::snippets/fips-snippet.adoc[]
 --
 
+* This release of the {descheduler-operator} updates the Kubernetes version to 1.29.
```

Comment:
The kube was rebased to 1.29 in 5.0.1. Yet, 5.0.1 images are grade D now. 5.0.2 updates the grade to A again. So the note about addressing several CVEs is still valid.

Reply:
@ingvagabund I do still have the note about the CVEs for 5.0.1 and 5.0.2. Here I'm just adding a mention about the kube bump too, to 5.0.2, since I started a precedent of it in KDO 5.1.1.

Right now it's only the KDO 5.1.1 that ONLY lists the kube bump and no CVEs addressed. Is this still accurate, or do we need need to add that 5.1.1 addressed any CVEs?

Reply:
5.1.0 -> 5.1.1 bumps image grades from C to A.

Reply:
Okay thanks, added that CVE line back in